web/client/src/views/Number.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <b-container v-if="isLookup" class="border p-4 mb-3">
+      <h3 class="text-center">Information</h3>
+      <b-container>
+        <b-row v-for="(value, name) in localData" :key="name" align-v="center">
+          <h5 class="text-capitalize m-0 mr-4">{{ name }}:</h5>
+          <p class="m-0">{{ value }}</p>
+        </b-row>
+      </b-container>
+    </b-container>
+
+    <b-container v-if="isLookup" class="border p-4">
+      <h3 class="text-center">Scanners</h3>
+      <Scanner
+        v-for="(scanner, index) in scanners"
+        :key="index"
+        :name="scanner.name.charAt(0).toUpperCase() + scanner.name.slice(1)"
+        :scanId="scanner.name"
+      />
+    </b-container>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { mapMutations, mapState } from "vuex";
+import { formatNumber, isValid } from "../utils";
+import Scanner from "../components/Scanner.vue";
+import axios, { AxiosResponse } from "axios";
+import config from "@/config";
+
+interface ScannerObject {
+  name: string;
+  description: string;
+}
+
+interface Data {
+  loading: boolean;
+  isLookup: boolean;
+  scanners: Array<ScannerObject>;
+  localData: {
+    valid: boolean;
+    raw_local: string;
+    local: string;
+    e164: string;
+    international: string;
+    countryCode: number;
+    country: string;
+    carrier: string;
+  };
+}
+
+export type ScanResponse<T> = AxiosResponse<{
+  success: boolean;
+  result: T;
+  error: string;
+}>;
+
+export default Vue.extend({
+  components: { Scanner },
+  computed: {
+    ...mapState(["number"]),
+    ...mapMutations(["pushError"]),
+  },
+  data(): Data {
+    return {
+      loading: false,
+      isLookup: false,
+      scanners: [],
+      localData: {
+        valid: false,
+        raw_local: "",
+        local: "",
+        e164: "",
+        international: "",
+        countryCode: 33,
+        country: "",
+        carrier: "",
+      },
+    };
+  },
+  mounted() {
+    this.runScans();
+  },
+  methods: {
+    async getScanners() {
+      try {
+        const res = await axios.get(`${config.apiUrl}/v2/scanners`);
+
+        this.scanners = res.data.scanners.filter(
+          (scanner: ScannerObject) => scanner.name !== "local"
+        );
```

Comment:
Good, note the local scan will be removed in the future, may be worth adding a TODO note here to remove this later